### ISSUE TITLE
Incorrect handling of \n in messages

### ISSUE THREAD

**Issue by megapctr** [i18next/i18next-parser#37]
I noticed that strings with `\n` get converted to `\\n` in the output files :(

**Comment by karellm** [comment]
Can you please detail your issues, provide reproducible step, example, failing tests... This is an OS project, open a PR or at the very least make the job of maintainers easier.

**Comment by megapctr** [comment]
Of course, I'd be happy to make some PRs when I have some free time.

About the issue, there's not much to add. When you have `i18n.t("Hello\nWorld!")` in your input files and run i18next-parser, the output json will look like this:
```json
{
  "Hello\\nWorld!": ""
}

**Comment by megapctr** [comment]
Here's how I use i18next-parser
```js
vfs.src(sourceFiles)
  .pipe(i18next({
    functions: ['__'],
    locales: languages,
    writeOld: false,

    // No namespaces and keys
    namespaceSeparator: '\t\t',
    keySeparator: '\t\t',
  }))
  .pipe(vfs.dest(partialResultsDir))
  .on('end', processJsonFiles);
```

**Comment by megapctr** [comment]
I've written a [test which confirms this bug](https://github.com/megapctr/i18next-parser/commit/321f2891e15da9fea21de210f2735ce454911595), but I couldn't decipher the regexes to find out what's causing the problem.

**Comment by danieljuhl** [comment]
Could we close this issue? I think it is fixed in #42

**Comment by megapctr** [comment]
I don't use this project anymore, but if it's fixed then yeah
